nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -112,11 +106,71 @@ public void testMetadataProcessing() throws Exception
     verifyTextGzipContent(is(equalTo(agricolae121Content)), contentAfterDelete);
   }
 
-  private void uploadPackage(String... names) throws IOException {
+  @Test
+  public void testDeleteAssetWithComponent() {
```

Comment:
I woulld rename this to something like: `testDeletingREmainingAssetAlsoDeletesComponent`

Reply:
Done in 6ef47d8db038b9f39d05981d3de9c28952390dbb